Dockerfile.downstream

```diff
@@ -2,6 +2,9 @@
 ARG TARGETARCH=amd64
 ARG COMMIT
 
+# Make kubectl & oc scripts available for copy
+FROM brew.registry.redhat.io/openshift4/ose-cli-rhel9:v4.17.0-202412032103.p0.g13001b0.assembly.stream.el9 as ose-cli
```

Comment:
we should have 4.18 images

Reply:
Sure, I'll take a look when I will have a working URL. For now I'm getting a not authorized

Reply:
@OlivierCazade seems like Konflux is passing on the image now but have a new error:
```
Failure snippet:
task build-container has the status "Failed":
File:     /opt/app-root/src/pkg/reconciler/taskrun/taskrun.go
  Line:     460
```

Could you PTAL ?

Reply:
```
network-obser1e43682853f1a139e10e4d4475d7ce84-build-container-1[Duration: 24 minutes 59 seconds]
NETWORK-OBSER1E43682853F1A139E10E4D4475D7CE84-BUILD-CONTAINER-1

task build-container has failed: "step-build" exited with code 1
[prepare] 2024/12/17 10:16:52 Entrypoint initialization

[place-scripts] 2024/12/17 10:16:53 Decoded script /tekton/scripts/script-1-8zf45
[place-scripts] 2024/12/17 10:16:53 Decoded script /tekton/scripts/script-2-8999l
[place-scripts] 2024/12/17 10:16:53 Decoded script /tekton/scripts/script-3-rxmpk
[place-scripts] 2024/12/17 10:16:53 Decoded script /tekton/scripts/script-4-cjnhj
[place-scripts] 2024/12/17 10:16:53 Decoded script /tekton/scripts/script-5-vdf6q
[place-scripts] 2024/12/17 10:16:53 Decoded script /tekton/scripts/script-6-stqxn

[use-trusted-artifact] Using token for quay.io/redhat-user-workloads/ocp-network-observab-tenant/netobserv-operator/network-observability-cli-container
[use-trusted-artifact] Restored artifact quay.io/redhat-user-workloads/ocp-network-observab-tenant/netobserv-operator/network-observability-cli-container@sha256:923c68f58baf5c5de5ee5cef899df2c81dc59895307e4897b80819e453b5cd1b to /var/workdir/source
[use-trusted-artifact] Using token for quay.io/redhat-user-workloads/ocp-network-observab-tenant/netobserv-operator/network-observability-cli-container
[use-trusted-artifact] Restored artifact quay.io/redhat-user-workloads/ocp-network-observab-tenant/netobserv-operator/network-observability-cli-container@sha256:201013b1a3d4977ac0ac1cd8d2797ba9c8262b8cfcfab338ad3f42c1c35ac4dd to /var/workdir/cachi2
[use-trusted-artifact] 

[build] mkdir -p ~/.ssh
[build] if [ -e "/ssh/error" ]; then
[build]   #no server could be provisioned
[build]   cat /ssh/error
[build]   exit 1
[build] fi
[build] Error allocating host: timed out waiting for instance address
[build] 
[build] Context info:
[build]   Platform: linux/arm64
[build]   File:     /opt/app-root/src/pkg/reconciler/taskrun/taskrun.go
[build]   Line:     460
[build] 

container step-build has failed  : [{"key":"StartedAt","value":"2024-12-17T10:17:05.341Z","type":3}]
```